tests/cgrep_test.py

```diff
@@ -0,0 +1,535 @@
+# Copyright 2011 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for cgrep.
+
+   Order doesn't matter for the purposes of these tests, so many
+   actual and expected results are sorted/sets to prevent issues relating to the
+   order in which items are returned."""
+
+import unittest
+import sys
+import argparse
+sys.path.append('../')
+from lib import naming
+from lib import nacaddr
+from cgrep import get_ip_parents
```

Comment:
I initially made the tests with just one import, and then it grew to 2, and so on, and I just kept repeating over time as more functions were added and tests needed to be made. I think it's better that these all be imported in a single statement though.